modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -10,47 +10,63 @@
 
 The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods.
 The pods contribute to the resource consumption in CPU and memory limits and requests.
-Learn how to calculate resources, such as memory and CPU, required to run {prod}.
 
-.Procedure
+To learn how to calculate resources such as memory and CPU that are required to run {prod}, you can use a provided example devfile.
```

Comment:
Moving it below makes it less-straight forward since you need devfile to calculate only workspace req